django/template/base.py

```diff
@@ -67,6 +67,7 @@
 from django.utils.timezone import template_localtime
 from django.utils.translation import gettext_lazy, pgettext_lazy
 
+from ..utils.deprecation import RemovedInDjango70Warning
```

Comment:
This project's [style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#imports) avoids multi-dot relative imports.